lib/model.js

```diff
@@ -2598,12 +2593,14 @@ class Model {
       }
       if (options.individualHooks) {
         await Promise.all(instances.map(async instance => {
-          const individualOptions = _.clone(options);
+          const individualOptions = {
+            ...options,
+            validate: false,
+            hooks: true
+          };
           delete individualOptions.fields;
```

Comment:
maybe this can leverage a `omit` on the options, bc delete is slow

Reply:
`omit` creates a new object entirely which is MUCH slower.

Reply:
I'd agree if this was more than 2-3 properties but for a single one cloning the entire object just isn't worth it.